import-export-cli/cmd/addApi.go

```diff
@@ -191,70 +226,67 @@ func createAPI(name string, namespace string, configMapName string, replicas int
 	//check interceptors dir is not empty
 	file, err := os.Open(interceptorsPath)
 	if err != nil {
 		utils.HandleErrorAndExit("cannot open interceptors Dir", err)
-		return ""
 	}
 	defer file.Close()
-	_, err = file.Readdir(1)
-	if err == nil {
-		//creating kubernetes configmap with interceptors
-		fmt.Println("creating configmap with ballerina interceptors")
-		errConfInt := createConfigMapWithNamespace(configMapName, interceptorsPath, namespace, operation)
-		if errConfInt != nil {
-			utils.HandleErrorAndExit("Error creating configmap for interceptors", err)
-		}
-		return configMapName
+	if _, err = file.Readdir(1); err != nil {
+		return
+	}
+
+	//creating kubernetes configmap with interceptors
+	fmt.Println("creating configmap with ballerina interceptors")
+	if err := createConfigMapWithNamespace(configMapName, interceptorsPath, namespace, operation); err != nil {
+		utils.HandleErrorAndExit("Error creating configmap for interceptors", err)
 	}
-	return ""
 }
 
-func handleJavaInterceptors(path string, operation string, namespace string, flagApiName string) []string {
+func handleJavaInterceptors(path string, operation string, namespace string, cmPrefixName string) []string {
 	var interceptors []string
 	var javaInterceptorsConfNames []string
 	//get interceptors if available
-	interceptorsPath := filepath.Join(path, filepath.FromSlash("libs"))
+	interceptorsPath := filepath.Join(path, "libs")
 	//check interceptors dir is not empty
-	file, err := os.Open(interceptorsPath)
-	if err != nil {
-		utils.HandleErrorAndExit("cannot open interceptors Dir", err)
+	if exists, err := utils.IsDirExists(interceptorsPath); !exists {
+		utils.HandleErrorAndExit("cannot open java interceptors Dir", err)
 	}
-	defer file.Close()
 
 	//get all jars in libs dir
 	errReadInterceptors := filepath.Walk(interceptorsPath, func(path string, info os.FileInfo, err error) error {
 		interceptors = append(interceptors, path)
 		return nil
 	})
 	if errReadInterceptors != nil {
-		utils.HandleErrorAndExit("cannot read interceptors in the libs", err)
+		utils.HandleErrorAndExit("cannot read interceptors in the libs", errReadInterceptors)
 	}
 	for _, filePath := range interceptors {
 		if filepath.Ext(filePath) == ".jar" {
 			//creating kubernetes configmap for each java interceptor
-			javaInterceptorsConfNames = append(javaInterceptorsConfNames, flagApiName + "-" + filepath.Base(filePath))
-			fmt.Println("creating configmap with java interceptor " + flagApiName + "-" + filepath.Base(filePath))
-			errConfInt := createConfigMapWithNamespace(flagApiName + "-" + filepath.Base(filePath), filePath, namespace, operation)
+			cmName := cmPrefixName + "-" + filepath.Base(filePath)
+			javaInterceptorsConfNames = append(javaInterceptorsConfNames, cmName)
+			fmt.Println("creating configmap with java interceptor " + cmName)
+			errConfInt := createConfigMapWithNamespace(cmPrefixName+"-"+filepath.Base(filePath), filePath, namespace, operation)
```

Comment:
Formatting issue.
**cmPrefixName+"-"+filepath.Base(filePath)** should be changed to **cmPrefixName + "-" +  filepath.Base(filePath)**